src/types.jl

```diff
@@ -1,13 +1,6 @@
 import Ratios: SimpleRatio
 import SaferIntegers: SafeInt
```

Comment:
Might be simplest to have, e.g., `Rational{Int16}` be the default, to keep the maintenance burden low (as well as time-to-load for the package). Then a user could use `Ratios{SafeInt}` if they so desire.